static/app/views/explore/tables/aggregatesTable.tsx

```diff
@@ -93,11 +90,20 @@ export function AggregatesTable({}: AggregatesTableProps) {
       <Table style={tableStyles}>
         <TableHead>
           <TableRow>
-            {fields.map((field, i) => (
-              <TableHeadCell key={i} isFirst={i === 0}>
-                {field}
-              </TableHeadCell>
-            ))}
+            {fields.map((field, i) => {
+              // Hide column names before alignment is determined
+              if (result.isPending) {
+                return <TableHeadCell key={i} isFirst={i === 0} />;
```

Comment:
Why are we still passing `isFirst` here?

Reply:
@Zylphrex  It's a required prop for `GridHeadCell` used to add a left border